### ISSUE TITLE
Suggestion: moving "new tab" button variants

### ISSUE THREAD

**Issue by MasterPetrik** [Bionus/imgbrd-grabber#790]
I'm using Grabber for several months and only few days ago I found "new tab" button...
Of cousre this is not so important problem, but anyway I want to suggest you to move "new tab" button somewhere else where it will be much easy to find.
**And important moment. About new opened tab position.**

## Current  "new tab" button position
The button too far from other tab interfase and placed in the end of tab bar, where it hard to find.
In current version "new tab" button opens new tab after current focused tab.
![2017-02-18_043121 win7 mastercore](https://cloud.githubusercontent.com/assets/22294259/23089143/9cbac704-f595-11e6-88c2-3410f67631fe.png)
## Variant 1  "new tab" button position
This variant is common for many programms and for all browsers, "new tab" button appears next to last opened tab.
**For this variant more common to open new tab after last tab. This is very obvious for user because new tab appears next to place where button was pressed.**
I guess it wouldn't work so good in grabber, considering having constant tabs "favourites" and "downloads" at the end of tabs order it will make button place not so obvious.
![2017-02-18_043121 win7 mastercore -](https://cloud.githubusercontent.com/assets/22294259/23089151/a9f5b870-f595-11e6-91be-f51b00c24da9.png)
if tabs doesn't fit in tab bar it become looks like current variant.
![2017-02-18_044356 win7 mastercore](https://cloud.githubusercontent.com/assets/22294259/23089251/097e1da4-f597-11e6-8c0d-80f73e3d0885.png)

## Variant 2  "new tab" button position
This variant is also common for tab-related programms. The "new tab" button fixed in front of all other tabs, at the left side of tab bar.
Its very easy to find, it's very close to search field and it more easy to implement than Variant 1.
**In this variant "new tab" button opens new tab after that button, so it becomes first tab. This is very obvious for user because new tab appears next to place where button was pressed.**
I'd prefer this variant.
![2017-02-18_043121 win7 mastercore - -](https://cloud.githubusercontent.com/assets/22294259/23089158/c6d6ec52-f595-11e6-85d4-163c55e4b412.png)

So I suggest you to think about it. 
Maybe you like current variant, or maybe you would change to one of these variants?

**Comment by barryanders** [comment]
Perhaps the tab widths could be adjusted to be responsive and keep the button on the far right.

Example: Safari does what I described above and the + button never moves.
![screen shot 2017-02-18 at 8 55 08 am](https://cloud.githubusercontent.com/assets/5648875/23094095/041a9ac4-f5b8-11e6-897a-f0f06d9d65c3.png)

**Comment by MasterPetrik** [comment]
Oh that extremely wide tabs... I wonder how it works when tabs doesn't fit in tab bar?
Maybe it looks like in Opera(lol):
![image](https://cloud.githubusercontent.com/assets/22294259/23094269/ad12b3c0-f606-11e6-80a4-16cc9f18392d.png)
this variant has its minuses too. for example, when you opened just a couple of tabs they are too wide for you to find tab labels instantly, moreover, yes, + button never moves, but with every open\close new tabs all tabs needs to be resized, and user should keep in mind new tab width and their positions. And "+" button in this variant still far from search field.

So every variant has its pluses and minuses, so it's you to decide what you prefer.

**Comment by barryanders** [comment]
At what point there are too many, they start cycling back and then you can scroll through them. Yeah, Opera's way is pretty ugly, so the responsive method would require extra work. Don't worry about bothering with it, lol. I hadn't thought through the headache that is with so many tabs.

![screen shot 2017-02-18 at 9 55 20 am](https://cloud.githubusercontent.com/assets/5648875/23094533/7a87a62c-f5c0-11e6-95f0-854f24e5592a.png)

**Comment by MasterPetrik** [comment]
Well, I guess, apple variant full of unnecessary graphical decorations, that are, in fact, no needed.
If you want an simple and elegant solution for tabs, wich doesn't fit in tab bar, then I can show you Firefox variant, well, most simple and effective: 
![image](https://cloud.githubusercontent.com/assets/22294259/23095283/55d80e90-f618-11e6-8b37-2c10f890b9b1.png)
Grabber uses the same method, and I guess that is very good choise.
But compare with classic version of this variant current Grabber variant in this case have own + and - : 
+ unlike classic variant, in grabber tab width is not constant and fabs can display full their labels(it is very useful)
- buttons "<" and ">" placed at the same place, on the right edge of tab bar, classic variant with "<" button on the left side and ">" button on the right side of tab bar are more convinient. 
![image](https://cloud.githubusercontent.com/assets/22294259/23095297/7437d32a-f618-11e6-9478-1fe812e1b873.png)
So I don't think that tab display variant need any changes, imho
My question is just about "+" button position.

**Comment by barryanders** [comment]
Yeah, I like the arrows. Also, one other way to handle tabs (not saying to do this, just talking about tabs) is to have a dropdown that shows all current open tabs. This is done in both Terminal and Photoshop.

Photoshop
![screen shot 2017-02-18 at 12 37 38 pm](https://cloud.githubusercontent.com/assets/5648875/23095862/2d3a711c-f5d7-11e6-9f3b-c9a477eebadc.png)


Terminal
![screen shot 2017-02-18 at 12 37 00 pm](https://cloud.githubusercontent.com/assets/5648875/23095866/351aab7c-f5d7-11e6-8ba6-135a8fb10a64.png)

**Comment by MasterPetrik** [comment]
> Yeah, I like the arrows. Also, one other way to handle tabs (not saying to do this, just talking about tabs) is to have a dropdown that shows all current open tabs. This is done in both Terminal and Photoshop.
I tell you more! Almost all exmples mentioned here has dropdown(firefox, Sublime Text, opera) and almost all programms which works with a lot of tabs, and yep, it's a very good addition to any of mentioned variants, but Grabber is not so tab-related programm, so it is not necessarily, but could be a good addition. But I guess it's not required for now.
 
> Also, one other way to handle tabs
 
Do you know? In grabber you have a very simple way to switching tabs - with mouse wheel, very useful!

**Comment by barryanders** [comment]
I just use ctrl+tab to switch tabs, crtl + t to open a new one, ctrl + w to close one, and pretty much never have more than two tabs open at once, so yeah, I agree this isn't important. I was just adding to the discussion.

**Comment by Penguin-Guru** [comment]
I agree that it would make more sense for the '+' button to be continuous with the tabs. I think it should be on whichever side the new tab would be added to, but it seems like new tabs are opened immediately right of the active tab so perhaps that's why the button is currently set aside.
